mkdocs/docs/configuration.md

```diff
@@ -339,25 +339,19 @@ catalog:
 
 | Key                 | Example                          | Description                                                                                        |
 | ------------------- | -------------------------------- | -------------------------------------------------------------------------------------------------- |
-| uri                 | <https://rest-catalog/ws>          | URI identifying the REST Server                                                                    |
+| uri                 | <https://rest-catalog/ws>        | URI identifying the REST Server                                                                    |
 | ugi                 | t-1234:secret                    | Hadoop UGI for Hive client.                                                                        |
```

Comment:
`ugi` should actually not be in REST catalog. its a hive catalog option that was accidentally added here https://github.com/apache/iceberg-python/commit/5209874cd4685427520b67b64188f5e9919f2816#diff-497e037708cc64870c6ba9372f6064a69ca1e74d65d6195dcee5a44851e8b47dR151